.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,55 @@
+name: Create Prerelease
+
+on:
+  workflow_dispatch:
+
+jobs: 
+  create_prerelease:
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+    - name: Install Ruby 3.2
+      uses: ruby/setup-ruby@7d546f4868fb108ed378764d873683f920672ae2 # tag v1.149.0
+      with:
+        ruby-version: 3.2
+
+    - name: Checkout code
+      uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # tag v3.5.0
+
+    - run: bundle
+
+    - name: Update files with new version
+      run: bundle exec rake newrelic:version:bump
+
+    - name: Update newrelic.yml (TODO)
+      run: echo "after hannahs work is merged"
+
+    - name: Set tag name
+      run: echo "prerelease_tag=$(bundle exec rake newrelic:version:current)-pre" >> $GITHUB_ENV
+
+    - name: Create pull request
+      run: |
+        git checkout -b "prerelease_updates_${{ env.prerelease_tag }}"
+        git add --all
+        git config --global user.email "ruby-agent@newrelic.com"
+        git config --global user.name "Github bot"
```

Comment:
We took the Java team's suggestion to hide our email for the release_notes.yml PR to avoid spam. That value is saved in the GitHub Actions secrets and can be accessed using `secrets.EMAIL`. 

I wonder if we want to set a constant for `user.name` too? The release_notes.yml workflow names the bot `newrelic-ruby-agent-bot`. Might be easier for searches. 🤷

Reply:
I'll update the email address. I wasn't using that same name because I'm not actually using the newrelic-ruby-agent-bot user to do anything, I just didn't know what other email address to use for the git config. I think since i'm not using the actual bot user, and just using the regular built in github bot, I think i'd lean more towards not sharing the name, just to avoid potential confusion.

Reply:
If I'm reading this correctly, these git config lines are for the "bump version" commit made on line 38. When I look in your test PR, that commit comes from the Ruby agent bot: https://github.com/newrelic/newrelic-ruby-agent/pull/2072/commits/959c55ac92d5dc5390cb9a54d05d72f2f5e4a725 

This surprised me because the PR isn't made by the Ruby agent bot but by the GitHub actions bot. However, it still used the Ruby agent bot's name anyway, so I'm not sure if `user.name` config matters if there's a GitHub user already assigned to the email address.

I could totally be looking at the wrong PR or something though! Regardless, I don't think it's a big deal and can totally stay as-is. Email obfuscation is more important.

Reply:
Ohhh weird! I didn't notice that. That must be because of the email being set to that, since I'm not actually using the bot at all. So maybe I should change the name then? Or maybe I should be using a different email address? I'm not sure what that email would be though. hmm.

Reply:
Personally, I like using the bot for this commit. So my vote would be to use the same username/email that we have for the release notes workflow. That way we're consistent about all the commits made by our team collectively (or I suppose, technically by no one?) Maybe James and Hannah would have other opinions?